### DISCUSSION TITLE
[Esame 31 Agosto 2021] Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#248] [Esami - Bertini @ Matematica]
![Screenshot 2023-11-13 194406](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/569b2cfc-fa36-4515-9698-0ad577be64b2)

**Comment by cornflexxx** [comment]
Chiamiamo $X_i$ con $i=1,..,6$ la variabile aleatoria $Geom(p_i)$ con  $p_i=\frac{6-i}{6}$,  che indica dopo quanti lanci si trova la $i$ - esima faccia del dado, quindi:

$$\mathbb{E}(\sum_{i=0}^5 X_i)=\sum_{i=0}^5 \mathbb{E}(X_i)=14.7$$

**Reply by Warcophyr** [reply]
@cornflexxx per come esce il risultato Pi non dovrebbe essere (6-i+1)/6
